cluster/gce/delete-stranded-load-balancers.sh

```diff
@@ -22,14 +22,31 @@ LIST=$(gcloud --project=${PROJECT} compute target-pools list --format='value(nam
 
 result=0
 for x in ${LIST}; do
-    if ! gcloud compute --project=${PROJECT} target-pools get-health "${x}" --region=${REGION} 2>/dev/null >/dev/null; then
-	echo DELETING "${x}"
-	gcloud compute --project=${PROJECT} firewall-rules delete "k8s-fw-${x}" -q
-	gcloud compute --project=${PROJECT} forwarding-rules delete "${x}" --region=${REGION} -q
-	gcloud compute --project=${PROJECT} addresses delete "${x}" --region=${REGION} -q
-	gcloud compute --project=${PROJECT} target-pools delete "${x}" --region=${REGION} -q
-        result=1
-    fi
+  if ! gcloud compute --project=${PROJECT} target-pools get-health "${x}" --region=${REGION} 2>/dev/null >/dev/null; then
```

Comment:
Can you add some comments here? Its hard to follow the logic.

Reply:
added comments.